<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replay</class>
 <widget class="QWidget" name="Replay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>Load log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pauseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="forwardButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Step 0/0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
